### ISSUE TITLE
1_6_nn_tutorial_jp.ipynb にて回避策が有効でない

### ISSUE THREAD

**Issue by morioka** [YutaroOgawa/pytorch_tutorials_jp#6]
貴重な翻訳をありがとうございます。

1_6_nn_tutorial_jp.ipynb にて
<pre>
日本語訳注：このチュートリアルの翻訳時上記のURLからデータセットをダウンロードすることができないという現象が発生しています（2020年11月24日-12月9日）。

ダウンロードに失敗した際は下記のコードを利用し、PyTorchのtorchvision.datasetsからデータセットを取得する方法をお試しください。
</pre>
となっている直後のセル番号4 を実行しましたが、2021/03/06 01:00時点で colab上で実行した際には HTTP 403レスポンスが返ってきます。

下記のように、いったん必要なファイルを data/MNIST/raw 以下に配置したあと、先ほどのセル番号4 を実行することで問題を回避できました。
<pre>
!wget http://yann.lecun.com/exdb/mnist/train-images-idx3-ubyte.gz -O data/MNIST/raw/train-images-idx3-ubyte.gz
!wget http://yann.lecun.com/exdb/mnist/train-labels-idx1-ubyte.gz -O data/MNIST/raw/train-labels-idx1-ubyte.gz
!wget  http://yann.lecun.com/exdb/mnist/t10k-images-idx3-ubyte.gz -O data/MNIST/raw/t10k-images-idx3-ubyte.gz
!wget http://yann.lecun.com/exdb/mnist/t10k-labels-idx1-ubyte.gz -O data/MNIST/raw/t10k-labels-idx1-ubyte.gz
</pre>

**Comment by YutaroOgawa** [comment]
@morioka さま

さらに、新たな別手法の提案をありがとうございます。

一つ別手法を記載しているので（Torchvisionから取得）、さらに追加は初心者の読者の混乱を招くため追記は避けたいと思います。

申し訳ございません。

ですが、このIssueはOpenにしておいていただき、Issueまで見るようなレベルの方には
「こうした手法もあるのか」

と分かるようにしたいです。

誠にありがとうございます！
